src/common/hooks/useInstance.js

```diff
@@ -37,6 +41,9 @@ const useInstance = (id) => {
     () => isSearchInstanceByIdLoading || isInstanceLoading,
     [isSearchInstanceByIdLoading, isInstanceLoading],
   );
+  const refetch = useCallback(() => {
+    refetchSearch().then(() => refetchInstance());
+  });
```

Comment:
Can this be done in Promise.all?

Reply:
Promise.all execute promises in parallel, but we need to keep sequence of calls